spec/features/normalized_download_spec.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Downloading normalzed files from POD', type: :feature do
+  let(:organization) { FactoryBot.create(:organization, code: 'best-org') }
+  let(:user) { FactoryBot.create(:user) }
+
+  before do
+    user.add_role :member, organization
+    login_as(user, scope: :user)
+
+    FactoryBot.create_list(:upload, 2, :binary_marc, organization: organization, stream: organization.default_stream)
+    FactoryBot.create_list(:upload, 2, :binary_marc_gz, organization: organization, stream: organization.default_stream)
+  end
+
+  describe 'Full dumps' do
+    before do
+      allow(Time.zone).to receive(:today).and_return('2020-01-01')
```

Comment:
![simpler times](https://2.bp.blogspot.com/-O_xRU8SoBXk/UXuSJQM6oAI/AAAAAAAALdA/JcqtXPQZItU/s1600/we-have-to-go-back.gif)

Reply:
![](https://media.giphy.com/media/7acBuI9xJomek/giphy.gif)